frontend/packages/dev-console/README.md

```diff
@@ -8,7 +8,8 @@ You'll need to install the `OpenShift Pipeline Operator` found in the Admin sect
 
 There is a helpful script to install a couple Pipelines found in the `yamls` folder.
 
-Link to the [Script](./yamls/pipelines/install_pipeline_mocks.sh)
+Link to the [Script](./yamls/pipelines/install_pipeline_mocks.sh) 
+(The above link is now deprecated)
```

Comment:
We should fix or remove information that are not valid anymore. IMHO, we don't want mark them as deprecated in our main README.

Reply:
even here the provided path does not exist